### PR TITLE
initial app setup, basic FBV routes

### PR DESCRIPTION
**Pull request by ccbass**
basic homepage and about page routes inside of a main app.

### CODE REVIEW

File: melodic-manatees/early_internet/main/templates/main/main-about.html [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
+    <section>
+        <h1>The project</h1>
+        <p>words go here about it</p>
+    </section>
+    <section>
+        <h1>The Team: Melodic-Manatees</h1>
+        <p>r00t ^blm (sadiq)#3476 - Lead</p>
+        <p>Demonkingj#4796</p>
+        <p>el_cocodrilo#7140</p>
+        <p>honorablebrah#5855</p>
```
Comment (edwardlee4948) [inline melodic-manatees/early_internet/main/templates/main/main-about.html:16]: <p>EM#2961</p> is missing

Comment (ccbass) [reply melodic-manatees/early_internet/main/templates/main/main-about.html:16]: Very strange, it's showing on my branch locally.  Let me recommit!

**Comment by ccbass** [comment]
Updated @eunminlee !

**Comment by sadiq-pasha** [comment]
approve

**Pull request merged**